style.css

```diff
@@ -0,0 +1,3273 @@
+@media screen{
+    body {
+        margin: 0;
+        padding: 0;
+        overflow-x: hidden;
+    }
+    
+    h1, h2, h3, h4, h5, p, ul, li{
+        margin: 0;
+        padding: 0;
+    }
+    
+    .css-1fe7a5q {
```

Comment:
code repetition is a bad practice. you can write things like  .class1,.class2 {
 width: 18px;
        padding-right: 13px;
}